content/en/docs/network_topology_aware_scheduling.md

```diff
@@ -286,17 +312,102 @@ data:
       - name: predicates
       - name: proportion
       - name: nodeorder
-      - name: binpack # Enable binpack plugin
-        arguments:
```

Comment:
Should also keep the binpack plugin as a bestpractice.

Reply:
Added